Sources/EmbraceCommonInternal/Protocols/SwizzleCache.swift

```diff
@@ -16,22 +16,29 @@ public class SwizzleCache {
     struct OriginalMethod: Hashable {
         let method: Method
         let baseClass: AnyClass
+        let swizzlerClass: String
```

Comment:
Why is this change necessary? Can you document what this String means?

Reply:
Sure, I can.
The change was needed because the cache is just saving the "BaseClass" (aka `NSURLSession`) and the method that we get from selector. To prevent any kind of problems when multiple classes swizzle the same selector, im also adding the name of the swizzler (for example, `DummyURLSessionInitWithDelegateSwizzler` or `URLSessionInitWithDelegateSwizzler`).